general/deny-external-users/rego/src.rego

```diff
@@ -0,0 +1,45 @@
+package denyexternalusers
+
+# RoleBinding Object
+violation[{"msg": msg}] {
+    obj := input.review.object
```

Comment:
I may be mistaken but all of these re-occurring assignments should be extractable out of block, leaving only eval logic in violation blocks.

Reply:
Valid point, but I will refactor this in a separate PR